genie-web/src/main/java/com/netflix/genie/web/tasks/job/JobCompletionService.java

```diff
@@ -154,6 +156,29 @@ void handleJobCompletion(final JobFinishedEvent event) throws GenieException {
         final Map<String, String> tags = MetricsUtils.newSuccessTagsMap();
 
         try {
+            /*
+             * Get the cluster and command chosen for the job for tagging metrics.
+             * Swallowing any exception in getting those as this step is not
+             * critical and should not show that the entire job failed
+             */
+            Cluster cluster;
+            Command command;
+
+            try {
+                cluster = this.jobSearchService.getJobCluster(jobId);
+            } catch (final Exception e) {
+                log.warn("Unable to tag metrics with cluster due to {}", e.getMessage(), e);
```

Comment:
slf4j does some of this for you, you can simplify this to: `log.warn("Unable to tag metrics with cluster", e);`